README.md

```diff
@@ -97,6 +97,7 @@ Game networking is a subset of computer networking that commonly covers data rep
   - [GameNetworkingSockets](https://github.com/ValveSoftware/GameNetworkingSockets) - Valve's internal (Steam) network transport layer for games.
   - [RakNet](https://github.com/facebookarchive/RakNet) - Full-featured and mature reliable UDP networking engine.
   - [yojimbo](https://github.com/networkprotocol/yojimbo) - Reliable UDP networking library for client/server games with dedicated servers.
+  - [TNL 1.5](https://github.com/nardo/torque_network_library_1_5) & [TNL 2.0](https://github.com/nardo/tnl2) - No longer maintained. Based on the TRIBES 1 & 2 and Torque networking stack and can be considered a reference implementation of the TRIBES networking paper.
```

Comment:
I'm happy to have TNL2 on the list, I think we should have a better format and description for the entry though.